dags/public-health/covid19/sync_covid_testing_data.py

```diff
@@ -0,0 +1,94 @@
+"""
+Pull data from MOPS COVID Dashboard and upload to ESRI
+"""
+import datetime
+
+import arcgis
+import pandas as pd
+
+from airflow import DAG
+from airflow.hooks.base_hook import BaseHook
+from airflow.operators.python_operator import PythonOperator
+
+
+def get_data(filename, workbook, sheet_name):
+    df = pd.read_excel(workbook, sheet_name=sheet_name, skiprows=1, index_col=0)
+    df = df.T
+    df = df.iloc[:, 1:3]
+    df.reset_index(level=0, inplace=True)
+    df.columns = ["Date", "Performed", "Test Kit Inventory"]
+    df["Date"] = pd.to_datetime(df["Date"], errors="coerce")
+    df = df.loc[df["Date"].dt.date < datetime.datetime.now().date()]
+    df.sort_values("Date", inplace=True)
+    cumulative = []
+    temp_value = 0
+    for i in df["Performed"]:
+        temp_value += i
+        cumulative.append(temp_value)
+    df["cumulative"] = cumulative
+    df.to_csv(filename, index=False)
+
+
+def update_arcgis(arcuser, arcpassword, arcfeatureid, filename):
+    filename = "/tmp/%s" % filename
+    gis = arcgis.GIS("http://lahub.maps.arcgis.com", arcuser, arcpassword)
+    gis_item = gis.content.get(arcfeatureid)
+    gis_layer_collection = arcgis.features.FeatureLayerCollection.fromitem(gis_item)
+    gis_layer_collection.manager.overwrite(filename)
+
+
+default_args = {
+    "owner": "airflow",
+    "depends_on_past": False,
+    "start_date": datetime.datetime(2020, 4, 16),
+    "email": [
+        "ian.rose@lacity.org",
+        "hunter.owens@lacity.org",
+        "brendan.bailey@lacity.org",
+    ],
+    "email_on_failure": True,
+    "email_on_retry": False,
+    "retries": 1,
+    "retry_delay": datetime.timedelta(days=1),
+}
+
+cron_string = "0 11-23 * * *"
```

Comment:
Circling back here: I just remembered that airflow uses UTC, so this won't execute at the intended time in PST.

Reply:
Is the solution then to create a new feature ID that uses UTC as timezone?

Reply:
This is entirely on the airflow time, which, IIRC, only uses UTC. The Esri side should be taken care of already, I think

Reply:
Got it. I think the appropriate conversion to UTC would be `* 0-6,18-23 * * *`

I'll update the script and push

Reply:
one other possible way to change this is how we do it in the `scooter-stat` DAG, which is to localize to PST?